### PR TITLE
Failing test cases regarding to #1325 #1337

### PR DESCRIPTION
**Pull request by vaheshadunts**

### CODE REVIEW

File: tests/Doctrine/Tests/Models/Pagination/Company.php [hunk 1]
```diff
@@ -27,4 +27,9 @@ class Company
      * @OneToOne(targetEntity="Logo", mappedBy="company", cascade={"persist"}, orphanRemoval=true)
      */
     public $logo;
-}
+
+    /**
+     * @OneToMany(targetEntity="Department", mappedBy="company", cascade={"persist"}, orphanRemoval=true)
+     */
+    public $departments;
+}
```
Comment (malukenho) [inline tests/Doctrine/Tests/Models/Pagination/Company.php]: EOL EOF

File: tests/Doctrine/Tests/Models/Pagination/Department.php [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    /**
+     * @Column(type="string")
+     */
+    public $name;
+
+    /**
+     * @ManyToOne(targetEntity="Company", inversedBy="departments", cascade={"persist"})
+     */
+    public $company;
+}
```
Comment (malukenho) [inline tests/Doctrine/Tests/Models/Pagination/Department.php]: same above

**Comment by zeroedin-bill** [comment]
I've rewritten these testcases into an upcoming PR. I've expanded the tests to check a wider variety of scenarios.

**Comment by zeroedin-bill** [comment]
Merged into #1353

**Comment by vaheshadunts** [comment]
closing this PR as @zeroedin-bill merged it into #1353

**Pull request closed without merging**